src/ViewArticle/ViewArticleStyles/ViewArticle.scss

```diff
@@ -0,0 +1,95 @@
+@import '../../commons/assets/variables/font.scss';
+@import '../../commons/assets/variables/colors.scss';
+
+.article_title{
+    color: #00a4d3;
+    font-family:Lato;
+    font-weight: light;
+    font-size: 40px;
+    margin-top:5px;
+}
+.myimage{
+    width: 90%;
+    margin-left: 5%;
+    margin-right: 5%;
+    height:259px;
+}
+
+.article_image {
+    width: 100%;
+    height:259px;  
+}
+.article-content {
+    width: 90%;
+    margin-left: 5%;
+    margin-right: 5%;
+    overflow: auto;
+}
+
+.article-content > p {
+    font-family: Source Sans Pro;
```

Comment:
@jennizalwango kindly consider calling this color from the variables you imported in line two of this file.

Reply:
okay let me work on this

Reply:
this is done @HabibSentongo